src/rust/ide/lib/view/src/graph_editor.rs

```diff
@@ -1176,6 +1191,18 @@ impl GraphEditorModel {
         }
     }
 
+    fn set_documentation_visibility(&self, is_vis: bool) {
+        if is_vis {
+            self.scene.remove_child(&self.doc_view);
+        } else {
+            self.scene.add_child(&self.doc_view);
+        }
+    }
+
+    pub fn is_doc_visible(&self) -> bool {
```

Comment:
why this is pub? 3rd party components should communicate with us via frp. Make it private pls